apps/marketplace/src/app/contact-us/contact-us.component.html

```diff
@@ -1 +1,21 @@
-<p>contact-us works!</p>
+<h1>Contact Us</h1>
+
+<h4>Mon - Fri 10:00 AM - 8:00 PM CET</h4>
+<h3>Send us a message and we’ll quickly get back to you :</h3>
+
+<div class="section">
```

Comment:
can you use mat-form-field here? then you could also style it with the appearance how it should render.

Reply:
Yeah will do the required changes. Tracked via:
https://github.com/blockframes/nft/issues/37

Will commit the functional page now.